space-plugins/nuxt-starter/README.md

```diff
@@ -61,6 +67,9 @@ Ensure that "Production" is the section that contains information about the prod
 
 <img src="./docs/open-extension.png" alt="Open the extension" width="200" />
 
+> [!NOTE]
+> If the authentication flow isn't working, please check if you have an ad blocker extension enabled in your browser and disable it.
```

Comment:
Hey @eunjae-lee, just to confirm: do you mean the `shield` or it would be actually the `ad blocker`?